spec/react_on_rails/support/shared_examples/base_generator_examples.rb

```diff
@@ -95,7 +95,7 @@
       Rails.application.config.assets.paths << Rails.root.join("client", "assets", "stylesheets")
       Rails.application.config.assets.paths << Rails.root.join("client", "assets", "images")
       Rails.application.config.assets.paths << Rails.root.join("client", "assets", "fonts")
-      Rails.application.config.assets.precompile += %w( generated/server-bundle.js )
+      Rails.application.config.assets.precompile += %w( generated/server-bundle.js react_on_rails )
```

Comment:
I'm using an old sprokets on another project and we have to add the file
individually if we want them precompiled.

On Tue, Dec 29, 2015 at 3:57 AM, Justin Gordon <notifications@github.com>
wrote:

> In spec/react_on_rails/support/shared_examples/base_generator_examples.rb
> <https://github.com/shakacode/react_on_rails/pull/172#discussion_r48530975>
> :
>
> > @@ -95,7 +95,7 @@
> >        Rails.application.config.assets.paths << Rails.root.join("client", "assets", "stylesheets")
> >        Rails.application.config.assets.paths << Rails.root.join("client", "assets", "images")
> >        Rails.application.config.assets.paths << Rails.root.join("client", "assets", "fonts")
> > -      Rails.application.config.assets.precompile += %w( generated/server-bundle.js )
> > +      Rails.application.config.assets.precompile += %w( generated/server-bundle.js react_on_rails )
>
> Maybe this is just the way that the new Sprockets expects such files?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/shakacode/react_on_rails/pull/172/files#r48530975>.
>